plot_types/basic/stairs.py

```diff
@@ -0,0 +1,26 @@
+"""
+==========
+stairs(x, y)
+==========
```

Comment:
Docs won't build if formatting is incorrect: `/home/circleci/project/doc/plot_types/basic/stairs.rst:24: WARNING: Title overline too short.`
```suggestion
============
stairs(x, y)
============
```